### PR TITLE
Bar 53 messaging

### PR DESCRIPTION
**Pull request by aodhanocathain**

### CODE REVIEW

**Comment by seangaff** [comment]
LGTM

**Review by seangaff — APPROVED** [review]
LGTM

File: frontend/navigation/HomeStack.tsx [hunk 1]
```diff
@@ -7,37 +7,42 @@ import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
...
+          else if (route.name === 'Items') {
+            iconName = focused ? 'pricetag' : 'pricetag-outline';
+          }
+          return <Ionicons name={iconName} size={size} color={color} />;
+        },
+      })}>
+      <Tab.Screen name='Marketplace' component={MarketplaceStack} />
+      <Tab.Screen name='Items' component={MyItemsStack} />
+      <Tab.Screen name="Messaging" component={MessagingStack} />
+      <Tab.Screen name='Profile' component={ProfileScreen} />
```
Comment (dan-whelan) [inline frontend/navigation/HomeStack.tsx]: Need to change this From Profile Screen to ProfileStack, it breaks the edit profile navigation

**Review by dan-whelan — CHANGES_REQUESTED** [review]
As per comment

**Review by dan-whelan — APPROVED** [review]
LGTM

**Pull request merged**